README.md

```diff
@@ -1 +1,72 @@
-# To-Do-list-list-structure
\ No newline at end of file
+# To Do List
+
+In this project, I built a simple HTML list of To Do tasks. This simple web page will be built using webpack and served by a webpack dev server.
+
+
+## Built With
+
+- HTMl
+- Webpack
+- CSS
+- Javascript
+
+## Live Demo (if available)
+
+## Getting Started
+
```

Comment:
- [ ] OPTIONAL
Also kindly include in your ReadMe the _Getting Started_ section,  instructions on how to **run** your application.